backend/__tests__/queries.test.js

```diff
@@ -0,0 +1,323 @@
+		const query = `{ 
+      archipelago(islanderId: 3){
+        id
+        name 
+        islands{
+          id
+          name
+          nativeFruit
+          islanders{
+            id
+            name
+          }
+        }
+      } 
+    }`;
+		const expectedResponse = {
+			data: {
+				archipelago: {
+					id: "1",
+					islands: [
+						{
+							id: "1",
+							islanders: [
+								{ id: "1", name: "Francis" },
+								{ id: "3", name: "Corrina" },
+							],
+							name: "Raftel",
+							nativeFruit: "Peaches",
+						},
+						{
+							id: "2",
+							islanders: [{ id: "2", name: "Riki" }],
+							name: "Montoya",
+							nativeFruit: "Apples",
+						},
+					],
+					name: "Covidian Montréal Consortium",
+				},
+			},
+		};
+
+		const res = await request(app).post("/graphql").send({ query: query });
+		const parsedRes = JSON.parse(res.text);
+
+		expect(res.statusCode).toEqual(200);
+		expect(parsedRes).toEqual(expectedResponse);
+	});
+
+	it("retrieves the proper islands full response", async () => {
+		const query = `{ 
+      islands{
+        id
+        name
+        nativeFruit
+        islanders{
+          id
+          name
+        }
+      }
+    }`;
+		const expectedResponse = {
+			data: {
+				islands: [
+					{
+						id: "1",
+						islanders: [
+							{ id: "1", name: "Francis" },
+							{ id: "3", name: "Corrina" },
+						],
+						name: "Raftel",
+						nativeFruit: "Peaches",
+					},
+					{
+						id: "2",
+						islanders: [{ id: "2", name: "Riki" }],
+						name: "Montoya",
+						nativeFruit: "Apples",
+					},
+					{
+						id: "3",
+						islanders: [{ id: "4", name: "Bawb" }],
+						name: "Syracuse",
+						nativeFruit: "Oranges",
+					},
+				],
+			},
+		};
+
+		const res = await request(app).post("/graphql").send({ query: query });
+		const parsedRes = JSON.parse(res.text);
+
+		expect(res.statusCode).toEqual(200);
+		expect(parsedRes).toEqual(expectedResponse);
+	});
+
+	it("retrieves the proper island full response by islandId", async () => {
+		const query = `{ 
+      island(islandId: 1){
+        id
+        name
+        nativeFruit
+        islanders{
+          id
+          name
+        }
+      }
+    }`;
+		const expectedResponse = {
+			data: {
+				island: {
+					id: "1",
+					islanders: [
+						{ id: "1", name: "Francis" },
+						{ id: "3", name: "Corrina" },
+					],
+					name: "Raftel",
+					nativeFruit: "Peaches",
+				},
+			},
+		};
+
+		const res = await request(app).post("/graphql").send({ query: query });
+		const parsedRes = JSON.parse(res.text);
+
+		expect(res.statusCode).toEqual(200);
+		expect(parsedRes).toEqual(expectedResponse);
+	});
+
+	it("retrieves the proper island full response by islanderId", async () => {
+		const query = `{ 
+      island(islanderId: 3){
+        id
+        name
+        nativeFruit
+        islanders{
+          id
+          name
+        }
+      }
+    }`;

```

Comment:
Just noticed that your queries are all missing some indentations! Do you have format on save enabled? It's my favourite for fixing alignment 🤤

Reply:
Yeah indentation is onSave, I think it's because of the +... added by github when you review it